x/multisig/types/signing.go

```diff
@@ -57,6 +57,10 @@ func (m SigningSession) ValidateBasic() error {
 		return fmt.Errorf("expires at must be >0")
 	}
 
+	if m.CompletedAt > m.ExpiresAt {
```

Comment:
Why is there no equivalent change in keygen session's `ValidateBasic`?

Reply:
Need to add `ValidateBasic` for keygen session